docs/data_sources.md

```diff
@@ -112,6 +112,22 @@ Full parameters:
 directive @stub(value: JSON!)
 ```
 
+## @mergeWithStub
```

Comment:
I'm not crazy about the name... I'm spitballing a bit here, but what if this directive was implemented as a parameter to `@stub`?
So you could do something like `field: SomeEntity! @stub(value: {asd: 70}, strategy: REPLACE/MERGE)` (Strategy being an enum, could also have DEEPMERGE to address @AleF83 's concerns)
Maybe renaming `@stub` could go along with this, to `@inline` or `@static`or something

Reply:
When we initially discussed making this directive we did consider just doing it as a parameter on `@stub`, but we dropped the idea because it sounds weird with the `stub` name, and changing the `stub` name would be a breaking change so it would complicate things and take more time.

We could just rename the `mergeWithStub` directive and absorb the stub functionality into it (with the `strategy` parameter), while marking the `@stub` directive as deprecated (and we can remove it at a later point). WDYT?